src/commands/openInPortal.ts

```diff
@@ -6,10 +6,10 @@
 import { openInPortal as uiOpenInPortal } from '@microsoft/vscode-azext-azureutils';
 import { IActionContext } from '@microsoft/vscode-azext-utils';
 import { ext } from '../extensionVariables';
+import { AppResourceTreeItem } from '../tree/AppResourceTreeItem';
 import { ResourceGroupTreeItem } from '../tree/ResourceGroupTreeItem';
-import { ResourceTreeItem } from '../tree/ResourceTreeItem';
 
-export async function openInPortal(context: IActionContext, node?: ResourceGroupTreeItem | ResourceTreeItem): Promise<void> {
+export async function openInPortal(context: IActionContext, node?: ResourceGroupTreeItem | AppResourceTreeItem): Promise<void> {
```

Comment:
Obviously this is a longer-term thing but actions like open in portal / reveal / remove / etc. are the perfect candidates for having just one command defined in the RG extension.

Reply:
No action needed, just wanted to put the comment down.